src/main/webapp/dev/allJsUnitTests.jsp

```diff
@@ -201,8 +165,8 @@
             </div>
             <div class="table table-responsive table-striped table-bordered">
                 <a class="btn btn-default btn-xs pull-right pull-down course-enroll-for-test"
-                    href="<%=data.getInstructorCourseEnrollLink(courseDetails.course.id)%>"
-                    title="<%=Const.Tooltips.COURSE_ENROLL%>"
+                    href=""
```

Comment:
Yes. It has to do with this file becoming a JSP, thus this previously-innocent looking block becomes something parseable, and unfortunately we do not (nor we need to) have a variable named `data` or importing `Const`.
This is a symptom of copy-pasting from JSP file without checking :p